controllers/src/test/scala/cromwell/pipeline/controller/ProjectControllerTest.scala

```diff
@@ -22,7 +22,7 @@ class ProjectControllerTest extends AsyncWordSpec with Matchers with ScalatestRo
 
   private val projectService = mock[ProjectService]
   private val projectController = new ProjectController(projectService)
-  private val stranger = TestUserUtils.getDummyUser()
+  private val stranger = TestUserUtils.getDummyUserWithCredentials()
```

Comment:
Why do we need user with credentials here?

Reply:
@myazinn I fixed